### PR TITLE
add 'conda install cudatoolkit' to dreambooth 'training on 16GB' example

### PR DESCRIPTION
**Pull request by 0xdevalias**
fixes https://github.com/huggingface/diffusers/issues/1207

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Hmm, 

I'm always a bit worried about giving clear instructions how to install CUDA as everybody does it differently and I think we should just leave it as "use bitsandbytes with GPU" and then it's a bit up to `bitsandbytes` to make sure everything works as expected no? 

@patil-suraj @pcuenca what do you think?

**Comment by patil-suraj** [comment]
Agree with @patrickvonplaten , we can't assume that everyone is using `conda`, I think here we could just point to `bitsandbytes` docs.  @0xdevalias would be  awesome if you could update the PR to instead add the link to `bitsandbytes`

**Comment by 0xdevalias** [comment]
> would be awesome if you could update the PR to instead add the link to `bitsandbytes`

@patil-suraj This PR is set to allow edits by maintainers. I think it might be best if you/one of the others updated the text as you like it to be, rather than having to back and forth to get it right. Alternatively, if you make a change suggestion through GitHub's PR review, I'm happy to click apply on it.

File: examples/dreambooth/README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ accelerate launch train_dreambooth.py \
 
 With the help of gradient checkpointing and the 8-bit optimizer from bitsandbytes it's possible to run train dreambooth on a 16GB GPU.
 
-Install `bitsandbytes` with `pip install bitsandbytes`
+Install `bitsandbytes` with `pip install bitsandbytes`, and it's required dependency `cudatoolkit` with `conda install cudatoolkit`.
```
Comment (patil-suraj) [inline examples/dreambooth/README.md:95]:
```suggestion
To install `bitandbytes` please refer to this [readme](https://github.com/TimDettmers/bitsandbytes#requirements--installation).
```

**Comment by patil-suraj** [comment]
Thanks @0xdevalias , just updated the instruction.

**Comment by StatsGary** [comment]
Yes this is still affecting me and links to the bitsandbytes setup. I have tried to resolve the issue, but I am still getting errors. 

I have raised an issue on the bitsandbytes site: 

`RuntimeError: 
        CUDA Setup failed despite GPU being available. Inspect the CUDA SETUP outputs aboveto fix your environment!
        If you cannot find any issues and suspect a bug, please open an issue with detals about your environment:
        https://github.com/TimDettmers/bitsandbytes/issues`

**Pull request merged**